service/src/main/java/bio/terra/catalog/schema.json

```diff
@@ -0,0 +1,192 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "Catalog Schema V1",
```

Comment:
Does it make sense to put the version in the title? Looking at example schemas at json-schema.org, it looks like schemas have names that reflect what the represent. In this case, this represents a `catalog dataset`, or maybe just `dataset` if the schema `id` is scoped to the catalog service.